systemd-udevd/pkg.yaml

```diff
@@ -17,13 +17,10 @@ steps:
         sha512: "{{ .systemd_sha512 }}"
     env:
       SOURCE_DATE_EPOCH: {{ .BUILD_ARG_SOURCE_DATE_EPOCH }}
+  - network: default
```

Comment:
why is this needed?

Reply:
pip3 is used to install ninja etc. will package those into tools later